### DISCUSSION TITLE
How to force update the app and hide *new update* dialog box?

### DISCUSSION THREAD

**Discussion by Riken-Shah** [cotes2020/jekyll-theme-chirpy#1245] [Q&A] [answered]
### Checklist

- [X] I'm willing to follow the [Contributing Guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

I want to hide this message and force update the app, 
<img width="351" alt="Screenshot 2023-09-18 at 5 53 03 PM" src="https://github.com/cotes2020/jekyll-theme-chirpy/assets/63820270/28b63d0a-26ce-4d4b-91d1-cc9f34dc75d4">

I am new to Jekyll, I looked into the code but couldn't find it. 

Can someone please help me with this? 

### Operations you have already tried

I looked into code, searched on google, GitHub issues.

**Comment by cotes2020** [comment, marked as answer]
This is the update strategy of PWA — silently update in the background > pop up notification > user manually confirm > update pages.

You can either turn off PWA feature or accept this notification.

**Reply by Riken-Shah** [reply]
Got it @cotes2020 .
Thanks a ton! 👏
